org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLCompletions.java

```diff
@@ -596,8 +596,11 @@ private void collectAttributeNameSuggestions(int nameStart, int nameEnd, Complet
 			Range range = getReplaceRange(nameStart, replaceEnd, completionRequest);
 			boolean generateValue = !isFollowedBy(text, nameEnd, ScannerState.AfterAttributeName,
 					TokenType.DelimiterAssign);
-			for (ICompletionParticipant participant : getCompletionParticipants()) {
-				participant.onAttributeName(generateValue, range, completionRequest, completionResponse);
+			

```

Comment:
Please remove this code here. XMLCompletions is for generic XML completion and do that in xsd extension. I have initialized that in XSDCompletionParticipant , see https://github.com/angelozerr/lsp4xml/commit/5c8bb414d42aec473475b4e86a2daa8b781526ba